data/json/furniture.json

```diff
@@ -3774,6 +3774,54 @@
       ]
     }
   },
+  {
+    "type": "furniture",
```

Comment:
Don't forget to add your description strings from the items to the furnitures as well.

Reply:
Ooops, definitely would have forgot that.